@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -1086,6 +1089,7 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                         );
                         if (texCoordType !== undefined) {
                             textureCoordinates.push(vertices[i + 2], vertices[i + 3]);
+                            normals.push(...tempVertNormal.toArray());
```

Comment:
As discussed, normals should only be generated if elevation is enabled

Reply:
done.